crates/devolutions-agent-shared/src/date_version.rs

```diff
@@ -18,6 +18,12 @@ pub struct DateVersion {
     pub revision: u32,
 }
 
+impl DateVersion {
+    pub fn to_string_without_revision(&self) -> String {
```

Comment:
naming (non-blocking): What about this naming? Feel free to ignore if you prefer the current one!
```suggestion
    pub fn fmt_without_revision(&self) -> String {
```